### PR TITLE
fix: Markdownlint cleanup for ES

### PR DESCRIPTION
**Pull request by nschonni**
Cleanup after conversion and removing link titles left over from English

### CODE REVIEW

**Comment by nschonni** [comment]
@Graywolf9 did you want this to broken up into smaller PRs?

**Comment by Graywolf9** [comment]
> @Graywolf9 did you want this to broken up into smaller PRs?

Thank you so much @nschonni maybe it's not necesseary for this PR, almost all the changes are spaces and things like that, isn't it? like the other PRs so I can review easily this kind of changes

File: files/es/learn/common_questions/what_is_a_web_server/index.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Por ejemplo, para producir las páginas web que finalmente ves en tu navegador,
 
 ## Aprendizaje activo
 
-_No hay aprendizaje activo disponible. [](https://developer.mozilla.org/es/docs/MDN/Getting_started) **Por favor, considere colaborar.**_
+_No hay aprendizaje activo disponible. **Por favor, considere colaborar.**_
```
Comment (Graywolf9) [inline files/es/learn/common_questions/what_is_a_web_server/index.md:56]: Improving to not lost the link

```suggestion
_No hay aprendizaje activo disponible. [**Por favor, considere colaborar.**](/es/docs/MDN/Getting_started)_
```

File: files/es/web/css/-webkit-box-reflect/index.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ translation_of: Web/CSS/-webkit-box-reflect
 
 ## Resumen
 
-`La propiedad `[CSS ](/es/docs/Web/CSS)`-webkit-box-reflect` permite reflejar el contenido del elemento en una dirección específica.
+`La propiedad`[CSS](/es/docs/Web/CSS) `-webkit-box-reflect` permite reflejar el contenido del elemento en una dirección específica.
```
Comment (Graywolf9) [inline files/es/web/css/-webkit-box-reflect/index.md:15]:
```suggestion
`La propiedad` [CSS](/es/docs/Web/CSS) `-webkit-box-reflect` permite reflejar el contenido del elemento en una dirección específica.
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much for your help!

**Pull request merged**